dateparser_data/supplementary_language_data/date_translation_data/ru.yaml

```diff
@@ -56,7 +59,34 @@ relative-type:
         - послезавтра
 
 simplifications:
-    - ^час: 1 час
-    - ^минуту: 1 минуту
-    - ^секунду: 1 секунду
+    - од(на|ну|ни|ной|ин): '1'
+    - дв(а|е|ое|ух): '2'
+    - пар[ауы]: '2'
+    - три: '3'
+    - четыре: '4'
+    - пять: '5'
+    - шесть: '6'
+    - семь: '7'
+    - восемь: '8'
+    - девять: '9'
+    - десять: '10'
+    - одиннадцать: '11'
+    - двенадцать: '12'
+    - пятнадцать: '15'
+    - двадцать: '20'
+    - тридцать: '30'
+    - сорок: '40'
+    - пятьдесят: '50'
```

Comment:
@noviluni this is probably closely related to the https://github.com/scrapinghub/number-parser project functionality

Reply:
It's ok to add these numbers. We will probably remove them when integrating `number-parser` (https://github.com/scrapinghub/dateparser/pull/711), but for now, it will be useful to have them here :)